lib/live_ex_webrtc/publisher.ex

```diff
@@ -220,115 +261,221 @@ defmodule LiveExWebRTC.Publisher do
-              />
-            </div>
-            <div class="flex gap-2.5 items-center">
-              <label for="lex-fps">FPS</label>
-              <input
-                type="text"
-                id="lex-fps"
-                value="30"
-                class="rounded-lg disabled:text-gray-400 disabled:border-gray-400 focus:border-brand focus:outline-none focus:ring-0"
-              />
+          <div
+            id="lex-video-settings-wrapper"
+            class="cursor-pointer bg-brand/10 rounded-md border border-brand/30"
+          >
+            <div
+              phx-click={
+                Phoenix.LiveView.JS.toggle_class("hidden",
+                  to: "#lex-video-settings"
+                )
+                |> Phoenix.LiveView.JS.toggle_class("rotate-180",
+                  to: "#lex-video-settings-wrapper .chevron"
+                )
+              }
+              class="px-4 py-3 flex items-center justify-between"
+            >
+              <div class="font-bold text-[#0d0d0d] py-2.5">Video Settings</div>
+              <div class="chevron transition-all duration-300">
+                <.icon name="hero-chevron-down" />
+              </div>
             </div>
-            <div class="flex gap-2.5 items-center">
-              <label for="lex-bitrate">Max Bitrate (kbps)</label>
-              <input
-                type="text"
-                id="lex-bitrate"
-                value="1500"
-                class="rounded-lg disabled:text-gray-400 disabled:border-gray-400 focus:border-brand focus:outline-none focus:ring-0"
-              />
+            <div
+              id="lex-video-settings"
+              class="hidden transition-all duration-700 text-[#606060] flex flex-col gap-6 px-4 pb-3"
+            >
+              <div id="lex-video-static" phx-update="ignore" class="flex flex-col gap-6">
+                <div id="lex-resolution" class="flex gap-2.5 items-center">
+                  <label for="lex-width">Width</label>
+                  <input
+                    type="text"
+                    id="lex-width"
+                    value="1280"
+                    class="rounded-lg disabled:text-gray-400 disabled:border-gray-400 focus:border-brand focus:outline-none focus:ring-0"
+                  />
+                  <label for="lex-height">Height</label>
+                  <input
+                    type="text"
+                    id="lex-height"
+                    value="720"
+                    class="rounded-lg disabled:text-gray-400 disabled:border-gray-400 focus:border-brand focus:outline-none focus:ring-0"
+                  />
+                </div>
+                <div class="flex gap-2.5 items-center">
+                  <label for="lex-fps">FPS</label>
+                  <input
+                    type="text"
+                    id="lex-fps"
+                    value="24"
```

Comment:
Just for the time being?
```suggestion
                    value="30"
```